bigquery_etl/metadata/validate_metadata.py

```diff
@@ -204,6 +204,35 @@ def validate_shredder_mitigation(query_dir, metadata):
     return True
 
 
+def validate_col_desc_enforced(query_dir, metadata):
+    """Check schemas with enforce_col_desc = True comply with requirements."""
+    # If column descriptions need to be enforced:
+    if metadata.enforce_col_desc is True:
```

Comment:
If you reverse this conditional you can return early when the new property isn't set and reduce the indentation of the rest of the function.

Also, I think it's more idiomatic Python to simple rely on the "truthiness" of values rather than using explicit comparisons like `is True` or `is not True`.